src/vmm/src/builder.rs

```diff
@@ -233,6 +233,13 @@ fn create_vmm_and_vcpus(
         setup_interrupt_controller(&mut vm)?;
         vcpus = create_vcpus(&vm, vcpu_count, &exit_evt).map_err(Internal)?;
 
+        // Make stdout non blocking.
+        let flags = unsafe { libc::fcntl(libc::STDOUT_FILENO, libc::F_GETFL, 0) };
```

Comment:
This should be moved to an aux function and do it for aarch64 [here](https://github.com/firecracker-microvm/firecracker/blob/6b3df1bc957edd00d530c66a7fb68ca8d417dbba/src/vmm/src/builder.rs#L635) too.

Reply:
Done.